#! /bin/sh /usr/share/dpatch/dpatch-run
## allow-larger-buffers-dpatch by Sven Mueller <debian@incase.de>
## updated to 2.3.13 by Duncan Gibb <duncan.gibb@siriusit.co.uk>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Patch to allow larger buffers (especially for list of UIDs).
## DP: Patch taken and "backported" to 2.2 from:
## DP: http://cyrus.brong.fastmail.fm/patches/cyrus-longwords-2.3.3.diff
## DP: Simplified for 2.3.13 since maxquoted and maxword became run-time
## DP: options.  Now just doubles MAXLITERAL.

@DPATCH@
diff -urNad git~/imap/imapparse.c git/imap/imapparse.c
--- git~/imap/imapparse.c	2010-01-16 19:21:09.000000000 -0200
+++ git/imap/imapparse.c	2010-01-16 19:35:43.889091595 -0200
@@ -53,7 +53,7 @@
 #include "exitcodes.h"
 
 enum {
-    MAXLITERAL = INT_MAX / 20
+    MAXLITERAL = INT_MAX / 10
 };
 
 void freebuf(struct buf *buf)
